drivers/hmis/app/models/hmis/ce/match/expression/last_enrolled_days_calculator.rb

```diff
@@ -0,0 +1,45 @@
+# frozen_string_literal: true
+
+module Hmis::Ce::Match::Expression
+  # Calculates the number of days since a client's last enrollment ended.
+  # Returns 0 if the client is still enrolled (no exit date).
```

Comment:
Specify that WIP enrollments are counted. I think this is true because we don't join to project at all. Probably worth having a test for it, though. 
```suggestion
  # Calculates the number of days since a client's last enrollment ended.
  # Returns 0 if the client is still enrolled (no exit date), even if the currently open enrollment is WIP
```

Reply:
good point, I'll add a test for it